src/views/transaction/components/fee/fee.styles.js

```diff
@@ -15,10 +15,19 @@ const useFeeStyles = createUseStyles((theme) => ({
     fontSize: theme.spacing(2),
```

Comment:
Note: This comment refers to lines 5-8 above. Shame on github not letting me add comments on unmodified code.

I don't think we need the rules `display`, `alignItems`, `justifyContent` and `flexDirection` there, since we only have a paragraph element inside.